### ISSUE TITLE
[NFR] Allow absolute paths in view

### ISSUE THREAD

**Issue by Mech7** [phalcon/cphalcon#1698]
Currently I cannot do something like this:

```php
$view->setLayoutsDir(getcwd().'/app/views/layout/');
```
As it will be prefixed with the path from the view dir, this makes it tricky when working with multiple module's.

My idea was installing all modules with composer, as it is not much more then a namespace. And allow modules to be anywhere under \my\module or under \module so I would not know the module path.

Please allow absolute paths in the layouts dir, also It would be nice if multiple view paths are supported for multi module applications. The views can have an extra module name between the view controller dir. (similiar to zf2)

**Comment by kevas** [comment]
```
$di->set('view', function() {
    $view = new MyView();
    return $view;
});

class MyView extends \Phalcon\Mvc\View {
    
    protected function _engineRender($engines, $viewPath, $silence, $mustClean, \Phalcon\Cache\BackendInterface $cache = null) {

        $path = realpath(dirname($viewPath));
        if($path !== false && $path !== realpath(PUBLIC_PATH)) {
            $basename = basename($viewPath);
            $viewPathArr = explode('/', $viewPath);
            $viewPathArrSlice = array_slice($viewPathArr, 1, -1);
            $viewPath = str_repeat('../', count($viewPathArrSlice)) . implode('/', $viewPathArrSlice) . '/' . $basename;
        }

        parent::_engineRender($engines, $viewPath, $silence, $mustClean, $cache = null);
    }
}
```

Constant PUBLIC_PATH is a path directory where is index.php file.
And then you can write absolute path to the method setLayoutsDir - $view->setLayoutsDir('/var/www/web/app/views/layout/');

**Comment by andresgutierrez** [comment]
Implemented in the 2.1.x branch
